api/src/pcapi/core/offers/api.py

```diff
@@ -1824,3 +1826,67 @@ def update_used_stock_price(stock: models.Stock, new_price: float) -> None:
             event=first_finance_event,
             reason=finance_models.PricingLogReason.CHANGE_AMOUNT,
         )
+
+
+def _is_allocine(provider_id: int) -> bool:
+    allocine_products_provider_id = get_allocine_products_provider_id()
+    allocine_stocks_provider_id = get_provider_by_local_class("AllocineStocks").id
+    return provider_id in (allocine_products_provider_id, allocine_stocks_provider_id)
+
+
+def _update_product_extra_data(product: offers_models.Product, movie: offers_models.Movie) -> None:
+    if not product.extraData:
+        product.extraData = offers_models.OfferExtraData()
```

Comment:
Est-ce que ça ne serait pas plus simple d'écrire 
```suggestion
    product.extraData = product.extraData or offers_models.OfferExtraData()
```